docs/metrics.md

```diff
@@ -186,6 +186,9 @@ The total number of tx packets dropped on vNIC interfaces. Type: Counter.
 ### kubevirt_vmi_network_transmit_packets_total
 Total network traffic transmitted packets. Type: Counter.
 
+### kubevirt_vmi_node_cpu_affinity
+VMI CPU affinities to node physical cores count. Type: Gauge.
```

Comment:
Perhaps it would be clearer as 
Number of VMI CPU affinities to node physical cores. Type: Gauge.

Reply:
I like it, updated